src/Apps/Search/Routes/Artworks/Components/Filter/Filters/FilterContainer.tsx

```diff
@@ -140,28 +141,28 @@ export class FilterContainer extends React.Component<
 
                   <Spacer mb={2} />
 
-                  {this.props.children(filters)}
+                  {this.props.children(context)}
```

Comment:
It's possible that `context` doesn't need to be passed into `children` anymore. I haven't tracked that down yet.